spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/ChannelNode.java

```diff
@@ -21,21 +21,18 @@
 /**
  * @author Andy Clement
  * @author David Turanski
+ * @author Ilayaperumal Gopinathan
  */
 public class ChannelNode extends AstNode {
 
-	private ChannelType channelType;
+	private String nameComponent;
 
-	private List<String> nameComponents;
+	private List<String> labelComponents;
```

Comment:
It is just a `name` and there is no distinction between the `tap` and the `named` destination now.